model/jpa/src/main/resources/META-INF/jpa-changelog-3.2.0.xml

```diff
@@ -0,0 +1,235 @@
+            <column name="REALM_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_AUTH_CONFIG_REALM" tableName="AUTHENTICATOR_CONFIG">
+            <column name="REALM_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLIENT_REALM" tableName="CLIENT">
+            <column name="REALM_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLIENT_CLIENT_TEMPL_ID" tableName="CLIENT">
+            <column name="CLIENT_TEMPLATE_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLIENT_ATTR_CLIENT" tableName="CLIENT_ATTRIBUTES">
+            <column name="CLIENT_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLIENT_DEF_ROLES_CLIENT" tableName="CLIENT_DEFAULT_ROLES">
+            <column name="CLIENT_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLI_ID_PRV_MAP_ID_PRV" tableName="CLIENT_IDENTITY_PROV_MAPPING">
+            <column name="IDENTITY_PROVIDER_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLIENT_ID_PROV_MAP_CLIENT" tableName="CLIENT_IDENTITY_PROV_MAPPING">
+            <column name="CLIENT_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLIENT_NODE_REG_CLIENT" tableName="CLIENT_NODE_REGISTRATIONS">
+            <column name="CLIENT_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLIENT_SESSION_SESSION" tableName="CLIENT_SESSION">
+            <column name="SESSION_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLI_SES_AUT_STAT_CLI_SES" tableName="CLIENT_SESSION_AUTH_STATUS">
+            <column name="CLIENT_SESSION" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLI_SESS_NOTE_CLI_SESS" tableName="CLIENT_SESSION_NOTE">
+            <column name="CLIENT_SESSION" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLI_SES_PRT_MAP_CLI_SES" tableName="CLIENT_SESSION_PROT_MAPPER">
+            <column name="CLIENT_SESSION" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLI_SESS_ROLE_CLI_SESS" tableName="CLIENT_SESSION_ROLE">
+            <column name="CLIENT_SESSION" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLIENT_TEMPLATE_REALM" tableName="CLIENT_TEMPLATE">
+            <column name="REALM_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CLIENT_TEMPL_ATTR_TEMPL" tableName="CLIENT_TEMPLATE_ATTRIBUTES">
+            <column name="TEMPLATE_ID" type="VARCHAR(36)"/>
+        </createIndex>
+        <createIndex indexName="IDX_CL_USR_SESS_NOT_CL_SESS" tableName="CLIENT_USER_SESSION_NOTE">
+            <column name="CLIENT_SESSION" type="VARCHAR(36)"/>
```

Comment:
This index is unnecessary because there is already an index for primary key on CLIENT_USER_SESSION_NOTE(**CLIENT_SESSION**, NAME) named _constr_cl_usr_ses_note_. Please remove.